packages/core/addon/models/bard-request-v2/fragments/base.js

```diff
@@ -54,4 +55,18 @@ export default class Base extends Fragment.extend(Validations) {
     }
     return this.metadataService.getById(this.type, this.field, this.source);
   }
+
+  @computed('field', 'parameters.{}')
+  get canonicalName() {
+    const { field: metric, parameters } = this;
+
+    return canonicalizeMetric({
```

Comment:
Are we using this `canonicalizeMetric` in a general way? Do we need to rename this?

Reply:
Yes this should be renamed, added a TODO